lib/android-helpers.js

```diff
@@ -890,5 +891,51 @@ helpers.isEmulator = function isEmulator (adb, opts) {
 helpers.bootstrap = Bootstrap;
 helpers.unlocker = unlocker;
 
-export { helpers, SETTINGS_HELPER_PKG_ID, prepareAvdArgs, ensureNetworkSpeed };
+const CONSTRAINTS = {
+  reboot: {
+    isBoolean: true,
+  },
+  chromeDriverPort: {
+    isNumber: true,
+  },
+  chromedriverExecutable: {
+    isString: true,
+  },
+  suppressKillServer: {
+    isBoolean: true,
+  },
+};
+
+const DEFAULT_SERVER_ARGS = {
```

Comment:
I'd remove these defaults, since the default values for corresponding capabilities are anyway already set to these. Which means in case the corresponding arg is not set then we just skip it for good